task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -38,8 +38,9 @@ spec:
         . /utils.sh
         SNYK_TOKEN="$(cat /etc/secrets/snyk_token)"
         if [[ -z $SNYK_TOKEN ]]; then
-          echo "SNYK_TOKEN is not defined, please create the $(params.SNYK_SECRET) secret with the snyk_token key containing the Snyk token." | tee stdout.txt
-          make_result_json -r SKIPPED -t "Check skipped: SNYK_TOKEN is empty"
+          echo "SNYK_TOKEN is not defined. Could create the $(params.SNYK_SECRET) secret with the snyk_token key containing the Snyk token." | tee stdout.txt
+          note="Task $(context.task.name) is skipped: Task is skipped because SNYK_TOKEN is empty."
```

Comment:
SR: note="Task $(context.task.name) skipped: The `SNYK_TOKEN` is empty."

- any guidance on acceptable values for this token? or can it be anything?